server/resources/[VORP]/[homestead_essentials]/homestead_jobs/server/main.lua

```diff
@@ -0,0 +1,30 @@
+local VorpCore = {}
+
+TriggerEvent("getCore", function(core)
+    VorpCore = core
+end)
+
+--- @param source integer
+--- @param job string
+--- @param jobgrade integer
+VorpCore.addRpcCallback("homestead_jobs:validateJob", function(source, callback, job, jobgrade)
+    local jobsData = LoadResourceFile(GetCurrentResourceName(), "shared/jobs.json")
+    jobsData = json.decode(jobsData)
+    local exists = false
+    for job, grades in pairs(jobsData.Jobs) do
+        if job == job then
+            for grade, data in pairs(grades) do
```

Comment:
Is there a way to convert a list of entities into a table? Feels like a table would be more optimal than O(N^2)